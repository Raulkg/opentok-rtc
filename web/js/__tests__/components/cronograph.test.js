var cronograph = require('../../components/cronograph.js');
test('snapshot for cronograph component', () => {
  expect(cronograph).toMatchSnapshot();
});