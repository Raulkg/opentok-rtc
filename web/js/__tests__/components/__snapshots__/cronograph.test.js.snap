// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for cronograph component 1`] = `
Object {
  "Cronograph": Object {
    "init": [Function],
    "reset": [Function],
    "start": [Function],
    "stop": [Function],
  },
}
`;
